xskillscore/tests/test_probabilistic.py

```diff
@@ -19,24 +19,54 @@
     xr_threshold_brier_score,
 )
 
+DIMS = ['lon', 'lat', ['lon', 'lat'], None, []]
+
 
 @pytest.fixture
```

Comment:
All of these could be moved to `conftest.py` to clean this up and might be usable in other modules?

Reply:
Let's do conftest in another PR